lib/response.js

```diff
@@ -269,13 +209,11 @@ Response.prototype.link = function link(l, rel) {
  * @param    {Number} code http status code
  * @param    {Object | Buffer | Error} body the content to send
  * @param    {Object} headers  any add'l headers to set
- * @param    {Function} callback a callback for use with async formatters
- * @returns  {Object | undefined} when async formatter in use, returns null,
```

Comment:
You may have already planned to do this, but consider updating the `response.md` documentation to be consistent with the new JSDoc, just so you don't forget to do it later.